demo/lib/TransformationControls/addFooterControl.js

```diff
@@ -101,9 +101,13 @@ export default [
   null,
   (iframeWindow, iframeDocument, selectedValue, iframe) => {
     event.target.disabled = true
-    addFooterContainer(iframeWindow, iframeDocument)
-    addFooterMenu(iframeWindow, iframeDocument)
-    addFooterReadMore(iframeWindow, iframeDocument, iframe)
-    addFooterLegal(iframeWindow, iframeDocument)
+    if (iframeWindow.pagelib.c1) {
+      // TODO: footer transformations are not exposed in the abstraction layer
```

Comment:
correct, coming in #208

Reply:
Probably better to wait for the PR to be merged then.